front/scripts/create_test_workspaces.ts

```diff
@@ -0,0 +1,91 @@
+import { Authenticator } from "@app/lib/auth";
```

Comment:
Can we move this one to a `dev` sub folder and add an assertion that `NODE_ENV` is development so we don't run it in production 🙏